integration_test/third_party_apps_data/applications/jetty/metadata.yaml

```diff
@@ -0,0 +1,68 @@
+public_url: "https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/jetty"
+short_name: Jetty
+long_name: Jetty
+description: |-
+  The Jetty integration collects
+  access logs. Access logs are parsed into a JSON payload
+  focused on request details. 
+
+  For more information about Jetty, see
+  [The Jetty Docs](https://www.eclipse.org/jetty/documentation/jetty-11/operations-guide/index.html).
+minimum_supported_agent_version:
+  metrics: 2.9.0
+  logging: 2.9.0
+supported_app_version: ["11.x", "10.x", "9.4.x"]
+expected_logs:
+- log_name: jetty_access
+  fields:
+  - name: httpRequest.remoteIp
+    type: string
+    description: ''
+  - name: httpRequest.requestUrl
+    value_regex: \/forbidden.html\?something=something
+    type: string
+    description: ''
+  - name: httpRequest.protocol
+    value_regex: HTTP\/1.1
+    type: string
+    description: ''
+  - name: httpRequest.requestMethod
+    value_regex: GET
+    type: string
+    description: ''
+  - name: httpRequest.userAgent
+    value_regex: curl\/.*
+    type: string
+    description: ''
+  - name: httpRequest.responseSize
+    type: string
+    description: ''
+  - name: httpRequest.userAgent
+    type: string
+    description: ''
+  - name: httpRequest.status
+    type: string
+    description: ''
+- log_name: syslog
+  fields:
+  - name: jsonPayload.message
+    value_regex: '.*oejs\.Server:main: Started Server\@.*\{STARTING\}.*'
+    type: string
+    description: null
```

Comment:
All the other descriptions default to `''` instead of `null`. We should do that here too

Reply:
Gotcha, updated.